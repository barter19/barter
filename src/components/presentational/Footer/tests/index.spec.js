import { shallow } from '@vue/test-utils';
import FooterComponent from '../index';

const renderer = require('vue-server-renderer').createRenderer();

describe('Footer Component', () => {
  const ClonedComponent = shallow(FooterComponent);
  const { vm } = ClonedComponent;

  it('renders to a nice snapshot', () => {
    renderer.renderToString(vm, (err, str) => {
      expect(str).toMatchSnapshot();
    });
  });
});
