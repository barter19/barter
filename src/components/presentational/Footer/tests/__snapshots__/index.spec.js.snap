// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component renders to a nice snapshot 1`] = `
<footer class="footer">
  <router-link to="/">Home</router-link>
  <router-link to="examples">Examples</router-link>
  <a href="https://github.com/mrboomer/vuecharged-template/blob/master/README.md" target="_blank">Docs</a>
</footer>
`;
